blackbox/docs/sql/ddl/column_policy.txt

```diff
@@ -47,34 +53,50 @@ If a new column ``a`` was added with type ``boolean``, adding strings to this
 column will result in an error, except the string can be implicit casted to a
 ``boolean`` value.
 
-Examples::
+Here's an example::
 
-    cr> create table my_table12 (
+    cr> create table my_table (
     ...   title string,
     ...   author string
     ... );
     CREATE OK, 1 row affected (... sec)
 
 .. hide:
 
-    cr> drop table my_table12;
+    cr> drop table my_table;
     DROP OK, 1 row affected (... sec)
 
-which is exactly the same as::
+Which is exactly the same as::
 
-    cr> create table my_table13 (
+    cr> create table my_table (
     ...   title string,
     ...   author string
     ... ) with (column_policy = 'dynamic');
     CREATE OK, 1 row affected (... sec)
 
 .. hide:
 
-    cr> drop table my_table13;
+    cr> drop table my_table;
```

Comment:
the hidden drop table statement should be moved to the end of the file as we execute insert and show create table right below.